app/screens/conversion-flow/conversion-confirmation-screen.tsx

```diff
@@ -57,8 +57,13 @@ export const ConversionConfirmationScreen = ({
 
   const { data } = useConversionScreenQuery({ fetchPolicy: "cache-only" })
 
-  const usdWallet = data.me.defaultAccount.usdWallet
-  const btcWallet = data.me.defaultAccount.btcWallet
+  const usdWallet = data?.me?.defaultAccount.usdWallet
+  const btcWallet = data?.me?.defaultAccount.btcWallet
+
+  if (!data?.me || !usdWallet || !btcWallet) {
+    // TODO: handle errors and or provide some loading state
```

Comment:
note: because `fetchPolicy` is `cache-only`, I don't think there is any loading to consider.

Reply:
Gotcha, from the perspective of types I dont think that is represented